src/components/SecondaryPanes/XHRBreakpoints.css

```diff
@@ -82,11 +82,36 @@
 }
 
 .xhr-input-method {
-  display: none;
+  visibility: hidden;
+  margin-inline-start: 4px;
+  padding-block: 0;
+  padding-inline-start: 2px;
+  padding-inline-end: 14px;
+  border: none;
+  border-radius: 2px;
+  font: inherit;
+  color: inherit;
+  -moz-appearance: none;
+  appearance: none;
+  background-color: var(--toolbarbutton-background);
+}
+
+.xhr-input-container.focused::after {
+  content: "";
+  position: relative;
+  right: 25px;
+  top: 5px;
+  width: 10px;
```

Comment:
The original image is 8x8 pixel, and is sharp at that size. Can you use 8px here?
The mask-size can probably be `contain`.

Reply:
Okay, fixed!